next.config.js

```diff
@@ -26,5 +26,8 @@ module.exports = withOffline(
         },
       ],
     },
+    env: {
```

Comment:
Next.js __dirname doesn't actually point to the root dir of the project (or at least where the script was called from). This resolves the issue by storing __dirname in the `process.env.ROOT` variable.